textfinding_indikator5.py

```diff
@@ -0,0 +1,107 @@
+# this code is for TEXT FINDING - INDIKATOR 5
+# input is txt file from preprocess dokumen lama, nama instansi & judul from dokumen lama & baru
+
+import re
+import preprocess_dokbaru as dokbaru
+import preprocess_doklama as doklama
+from utility import *
+
+def txtreader(filename, lv, keyword):
+
+    file = open(f'cleaned_{filename}.txt', 'r')
+
+    idx = 0
+    result = []
+    
+    reg1 = f'(?:(layanan)?\s*?((pusat)\s*(data)))'
+    reg2 = f'(?:(pusat)?\s*?((data)\s*(center)))'
+
+    for line in file:
+        if (lv == 1):
+            if (re.search(reg1, line.lower(), re.IGNORECASE) or re.search(reg2, line.lower(), re.IGNORECASE)):
+                result.append([idx, line])
+
+        else:
+            if (re.search(reg1, line.lower(), re.IGNORECASE) or re.search(reg2, line.lower(), re.IGNORECASE)):
+                for key in keyword:
+                    if re.search(key, line.lower(), re.IGNORECASE):
+                        result.append([idx, line])
+
+        idx += 1
+
+    file.close()
+    return (result)
+
+def ceklvl(filename):
+    list_final = []
+
+    lvl1 = []
+    res1 = txtreader(filename, 1, lvl1)
+
+    if (not res1):
+        return ''
+
+    lvl2 = convert_keywords([
+        'organisasi perangkat daerah', 
+        'opd', 
+        'unit kerja', 
+        'pemerintah daerah', 
+        'perangkat daerah'])
+    res2 = txtreader(filename, 2, lvl2)
+
+    if (not res2):
+        return ''
```

Comment:
Check this comment here https://github.com/Tugas-Akhir-SPBE-IF-2019/ta-spbe-textfinding/pull/7/files/9281deef51004de3bc32dd4947f2b33504c09d63#r1141619220

This should return the lines found from lvl1 (the values in `res1` variable)